tests/GoDaddyClientTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Godaddy\V1\GetDomainAvailableRequest;
+use Vendasta\Godaddy\V1\GoDaddyClient;
+
+class GoDaddyClientTest extends TestCase
+{
+    public function testGetDomainAvailableWhenDomainIsTaken()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new GodaddyClient($environment);
+
+        $req = new GetDomainAvailableRequest();
+        $req->setDomain("google.com");
+
+        try {
+            $resp = $client->GetDomainAvailable($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::assertFalse(
```

Comment:
Rather than true == false, does [fail](https://stackoverflow.com/questions/4166522/how-to-force-a-failure-with-phpunit) work here?